shared/settings/index.js

```diff
@@ -57,7 +57,7 @@ const Connected = compose(
 // $FlowIssue lets fix this
 Connected.navigationOptions = {
   header: undefined,
-  title: 'Settings',
+  title: '',
```

Comment:
Would you mind adding a comment here explaining this is for mobile and we intentionally have no title here on mobile?

Reply:
done!